tests/driver_sdcard_spi/main.c

```diff
@@ -50,9 +50,7 @@ static int _init(int argc, char **argv)
 
     if (sdcard_spi_init(card, &sdcard_spi_params[0]) != 0) {
         puts("[FAILED]");
-        #if ENABLE_DEBUG != 1
-        puts("enable debugging in sdcard_spi.c for further information!");
-        #endif
+        puts("enable debugging in sdcard_spi.c for more information!");
```

Comment:
```suggestion
        if (!IS_ACTIVE(ENABLE_DEBUG)) {
            puts("enable debugging in sdcard_spi.c for more information!");
        }
```

Reply:
I think this if-statement was misplaced: there was/is no `ENABLE_DEBUG` in this file, so this check is unnecessary.

Reply:
Kept this as-is, with the above reasoning. For next PR.